src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -161,14 +161,6 @@ const Footer = BaseFooter.extend({
       Enums.UNIVERSAL_LINK_CHALLENGE
     ].includes(this.options.currentViewState.relatesTo.value.challengeMethod);
     if (isFallbackApproach) {
-      this.links = [
```

Comment:
Isn't really related to your changes but we shall try to override `links` method instead of `initialize`.
Check the `BaseFooter.js`

Reply:
Thanks @howardlee-okta and @haishengwu-okta